H/highs/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "HiGHS"
+version = v"0.1.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ERGO-Code/HiGHS.git", "30de0e3c2c977bd3c5bfd95e502c2ee041aa6525"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+mkdir -p HiGHS/build
+cd HiGHS/build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ..
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:i686, libc=:glibc),
+    Linux(:x86_64, libc=:glibc),
+    MacOS(:x86_64),
+    FreeBSD(:x86_64)
+]
+platforms = expand_cxxstring_abis(platforms)
```

Comment:
Oh, I missed this warning as well:
```
┌ Warning: /agent/_work/1/s/H/highs/build/i686-linux-gnu-cxx03/h6Ngs2fa/destdir/lib/libFortranHighs.so links to libgfortran!  This causes incompatibilities across major versions of GCC.  To remedy this, you must build a tarball for each major version of GCC.  To do this, immediately after your `platforms` definition in your `build_tarballs.jl` file, add the line:
│ 
│     platforms = expand_gfortran_versions(platforms)
└ @ BinaryBuilder /depot/packages/BinaryBuilder/RgjsE/src/auditor/compiler_abi.jl:59
```
```suggestion
platforms = expand_gfortran_versions(expand_cxxstring_abis(platforms))
```